app/tailwind.config.js

```diff
@@ -4,7 +4,18 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'background-blue':'#1A54A9',
+        'logo-yellow':'#debc6b',
+        'dark-yellow':'#977f48',
+        ogoyellow: "#debc6b",
+            logodarkyellow: "#977f48",
+            enquirygray: "rgb(209 213 219)",
+            footergray: "rgb(107 114 128)"
+      }
+        
+    },
```

Comment:
Store colors in a different file and then query from it. 
This way, we only have to change it at one place,  if needed.